setup.py

```diff
@@ -14,12 +14,15 @@
 # alternative forms of installing, as suggested by README.md).
 from setuptools import setup
 from setuptools.command.build_py import build_py
-from mypy.version import base_version
+from mypy.version import base_version, __version__
 from mypy import git
 
 git.verify_git_integrity_or_abort(".")
 
-version = base_version
+if 'bdist_wheel' in sys.argv[1:]:
```

Comment:
We'll need a similar exception for 'sdist'. See the misc/upload-pypy.py script, it runs `python3 setup.py bdist_wheel` and `python3 setup.py sdist`.

Otherwise this seems to work and solves my problem, so thanks!